go-controller/pkg/cni/ovs.go

```diff
@@ -198,22 +209,31 @@ func doPodFlowsExist(mac string, ifAddrs []*net.IPNet, ofPort int) bool {
 	return true
 }
 
-func waitForPodFlows(ctx context.Context, mac string, ifAddrs []*net.IPNet, ifaceName, ifaceID string, ofPort int) error {
+func waitForPodInterface(ctx context.Context, mac string, ifAddrs []*net.IPNet, ifaceName, ifaceID string, ofPort int, checkExternalIDs bool) error {
 	timeout := time.After(20 * time.Second)
 	for {
 		select {
 		case <-ctx.Done():
-			return fmt.Errorf("canceled waiting for OVS flows")
+			return fmt.Errorf("canceled waiting for OVS interface")
```

Comment:
maybe a better error is "canceled waiting for OVS port binding?"

Reply:
Sounds better indeed :)